### DISCUSSION TITLE
能否增加一个测试推送的功能呢

### DISCUSSION THREAD

**Discussion by ddaodan** [LmeSzinc/AzurLaneAutoScript#1611] [Ideas]
通过发送一条推送来测试json配置是否正确
不然就得自己想办法触发一次alas报错

另外希望推送是否成功能在日志里体现（如果已经有了的话就当我没看到吧）

**Comment by 18870** [comment]
![image](https://user-images.githubusercontent.com/22175295/188935380-efb5cd48-3a10-489e-8723-1e80a8b87226.png)
把游戏切换到邮件页面，找一个任务清空时间加入队列启动即可

**Comment by AkagiYui** [comment]
推送成功会提示 Push notify success
推送失败会抛出错误 Push notify failed

**Reply by ddaodan** [reply]
正好跑大世界的时候报错了，在日志里没找到，搜push啥也没有

**Reply by AkagiYui** [reply]
请提供你的配置

**Reply by ddaodan** [reply]
provider: smtp
host: smtp.qq.com
user: ***@qq.com
password: gth*********bfh
port: 465

好像是有问题，发不出去，准备换个别的
